packages/charts/chart-web-components/src/horizontal-bar-chart/horizontal-bar-chart.ts

```diff
@@ -71,8 +73,68 @@ export class HorizontalBarChart extends FASTElement {
     this.hydrateLegends();
   }
 
+  public enableTabbability() {
+    requestAnimationFrame(() => {
+      const legendContainer = this.shadowRoot?.getElementById('legend-div') as HTMLElement;
+      if (legendContainer) {
+        const legendButtons = tabbable.tabbable(legendContainer);
+        let currentLegendIndex = 0;
+
+        legendContainer.addEventListener('keydown', (event: KeyboardEvent) => {
```

Comment:
can we extract these event handlers into public class functions and use event binding on the element?